packages/webview_flutter/webview_flutter_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.3.0
+
+* Adds WebSettings methods: `AndroidWebViewController.setAllowContentAccess` and `AndroidWebViewController.setGeolocationEnabled`.
```

Comment:
Sorry after reviewing another PR I noticed this. I think this should be changed to

```suggestion
* Adds support for disabling content URL access within WebView and disabling the Geolocation API.
  See `AndroidWebViewController.setAllowContentAccess` and
  `AndroidWebViewController.setGeolocationEnabled`.
```

This is still ready for secondary review though.